scripts/autorelease.sh

```diff
@@ -14,9 +14,6 @@ assets_dir=$TEMPDIR/assets
 # Build release.nix
 nix-build release.nix -o "$TEMPDIR"/"$project" --arg timestamp "$(date +\"%Y%m%d%H%M\")"
 mkdir -p "$assets_dir"
-# Create archives with deb and rpm packages
-tar -cvzf "$assets_dir"/packages-deb.tar.gz --mode='u+rwX' -C "$TEMPDIR"/"$project" $(cd "$TEMPDIR"/"$project" && ls *.deb)
-tar -cvzf "$assets_dir"/packages-rpm.tar.gz --mode='u+rwX' -C "$TEMPDIR"/"$project" $(cd "$TEMPDIR"/"$project" && ls *.rpm)
 # Move these archives to assets
```

Comment:
Which archives are moved here?

Reply:
Initially there were `binaries-babylonnet` and `binaries-mainnet`, currently it's a single archive with binaries